css/section-2.css

```diff
@@ -0,0 +1,28 @@
+.who {
+  color: #b35efe;
+  font-family: "Open Sans", sans-serif;
+  font-size: 1rem;
+  font-weight: bolder;
+}
+#about-part-2 h1 {
+  color: #292c34;
+  font-family: "Open Sans", sans-serif;
```

Comment:
No need to write font family anwhere, it is already defined in `main.css`

Reply:
okay